Jenkinsfile

```diff
@@ -57,11 +57,19 @@ pipeline {
                 lock("BDD tests ${OPENSHIFT_API}")
             }
             stages {
+                stage("Checkout kogito-operator repo"){
```

Comment:
We may want to rename here something like `Retrieve community BDD features` and then in the script part
- checkout kogito-operator
- copy `kogito-operator/test/features` and `kogito-operator/test/scripts/examples` folders to `${WORKPSPACE}/test` folder

So we can keep the `make build-examples-images` and `make run-smoke-tests` commands into the next stages. wdyt ?

Not mandatory here, could be done in another PR

Reply:
Ideally I would like to get rid of example building, once we build those examples as part of nightly pipeline.
About the features, I don't have any strong opinion on whether move it to same structure as in community or reference the cloned folder.